yarn.lock

```diff
@@ -8,22 +8,22 @@
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
 "@grpc/grpc-js@^1.5.4":
-  version "1.6.7"
-  resolved "https://registry.yarnpkg.com/@grpc/grpc-js/-/grpc-js-1.6.7.tgz#4c4fa998ff719fe859ac19fe977fdef097bb99aa"
-  integrity sha512-eBM03pu9hd3VqDQG+kHahiG1x80RGkkqqRb1Pchcwqej/KkAH95gAvKs6laqaHCycYaPK+TKuNQnOz9UXYA8qw==
```

Comment:
This is a lot of changes to the dependencies. Did you go through all of these and verify that the licenses are ASF compatible and if the LICENSE/NOTICE needs to be updated.

Reply:
Yeah so the `moo` related packages were delete because we don't use them anymore they were just part of the repo we used to create this one. The `gRPC` related packages were deleted as we only used them when the `omega-edit` wrapper code was in this repo which it no longer is. These packages are still installed but as transitive dependencies instead of direct dependencies of `daffodil-vscode`. As for the 3 new packages added

```yaml
- @vscode/test-electron: MIT
- uuid: MIT
- wait-port: MIT
```

If I didn't add those already I will do so now

Reply:
Actually `@vscode/test-electron` should only be dev dependencies and not be packages I will fix that

Reply:
That was fixed and all the new bundled dependencies (plus their transitive dependencies) were added to the package LICENSE file as they were all MIT licensed.